testdata/project-v3-config/Makefile

```diff
@@ -35,7 +35,7 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
-	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.txt" paths="./..."
```

Comment:
the license is created for the base config and does require to be specific for Golang.
Then, has no need for it is called go.txt.

Reply:
The reason why the language needs to be part of the license file is because comments are different in other languages. For example, you could not use this license for Python operators.

Reply:
that is true. however, we can add by default a hack here which would be used by all with # to do the comments.

Reply:
We could even do a plugin that only scaffolds this file and chooses the right format depending on the language used, but as you said lets leave this outside of this PRs scope. Another reason to keep it outside of this PR is that this change is breaking while the rest of the can probably be non-breaking at least for the user.